src/include/platform/NetworkCommissioning.h

```diff
@@ -175,6 +175,25 @@ class BaseDriver
      */
     virtual NetworkIterator * GetNetworks() = 0;
 
+    /**
+     * @brief Returns the status of last network operations (i.e. scan and attach attempts.) If no such attempts are conducted,
+     * CHIP_ERROR_KEY_NOT_FOUND will be returned.
+     */
+    virtual CHIP_ERROR GetLastNetworkingStatus(Status & status) { return CHIP_ERROR_KEY_NOT_FOUND; }
```

Comment:
Don't these need to be implemented for all the drivers, not just Linux + Thread, if we're going to remove the shared cross-platform impl?

Reply:
Yes, other platforms should also implement this.